tos/tests/test_models.py

```diff
@@ -1,26 +1,26 @@
+from django.contrib.auth import get_user_model
```

Comment:
This breaks tests for Django 1.4; should either import get_user_model from from tos.models instead or do the same compatibility hack as in tos.models.